mandatory/2-function-creation.js

```diff
@@ -6,7 +6,10 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if ((typeof num === "number") && (num % 2 === 0) && (num <= 100))
+  return true;
```

Comment:
Well done, but what will be the returned value if the conditions have not been met ?